src/Lucid/Relations/Parser.js

```diff
@@ -21,17 +21,32 @@ const CE = require('../../Exceptions')
  */
 class RelationParser {
   /**
+   * Returns the relationships object
+   *
+   * @method _normalizeRelations
+   *
+   * @param {Object|String[]}             relations
+   *
+   * @return {Object}
+   */
+  _normalizeRelations (relations) {
+    return !Array.isArray(relations)
+      ? relations
+      : _.zipObject(relations, _.fill(new Array(relations.length), null))
```

Comment:
Do we really need `new Array` here. Also it will be simpler to write  `_.transform` here. Easy to understand

```js
_normalizeRelations (relations) {
    if (Array.isArray(relations)) {
        return relations
    }

    return _.transform(relations, (result, relation ) => {
        return (result[relation] = null)
    }, {})
}
```